index.html

```diff
@@ -80,7 +80,7 @@ <h1>The bank that does it all</h1>
 
 
                     <div class="btn-sec">
-                        <a href="/pages/sign up page">Get Started ></a>
+                        <a href="../Banka-project/pages/sign up page/index.html">Get Started ></a>
```

Comment:
I don't understand the link path here
I should be more like: `./pages/sign up page` No need to reference index.html. Just reference the folder
Secondly, change the folder name to something like `sign-up` and remove spaces. And there is no need to add pages at the end of the folder name since it's already in the pages folders.